application-admintools-default/src/test/java/com/xwiki/admintools/internal/data/identifiers/CurrentServerTest.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package com.xwiki.admintools.internal.data.identifiers;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Named;
+import javax.inject.Provider;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.xwiki.component.phase.InitializationException;
+import org.xwiki.test.junit5.mockito.ComponentTest;
+import org.xwiki.test.junit5.mockito.MockComponent;
+
+import com.xwiki.admintools.ServerIdentifier;
+import com.xwiki.admintools.configuration.AdminToolsConfiguration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * Unit test for {@link CurrentServer}
+ *
+ * @version $Id$
+ */
+@ComponentTest
+public class CurrentServerTest
+{
+    @InjectMocks
```

Comment:
Why not @InjectMockComponents ?

Reply:
If I use @InjectMockComponents I get this error: https://up1.xwikisas.com/#m8p2O-mnGDYgXeA2CrPA0g

Reply:
If you look at the logs, at the end there is the cause of this issue
```
Caused by: java.lang.NullPointerException

	at com.xwiki.admintools.internal.data.identifiers.CurrentServer.updateCurrentServer(CurrentServer.java:103)

	at com.xwiki.admintools.internal.data.identifiers.CurrentServer.initialize(CurrentServer.java:54)
```
It fails at initialization, because inside `updateCurrentServer` you have `supportedServers` as null.
So you actually found a bug with this test:)